e2e/README.md

```diff
@@ -0,0 +1,39 @@
+# TSGoLint E2E Tests
+
+Cross-platform end-to-end testing for TSGoLint using TypeScript and Vitest.
+
+## Setup
+
+```bash
+cd e2e
+pnpm install
+```
+
+## Running Tests
+
+```bash
+pnpm test
+```
+
+## Structure
+
+-   `fixtures/` - Test files organized by rule and general source files
+-   `snapshot.test.ts` - Main test file that runs TSGoLint and compares output
+-   `update-snapshot.ts` - Utility script to manually update the snapshot file
+-   `e2e_diagnostics.snap` - The expected diagnostics snapshot
```

Comment:
I don't think this is right - `e2e_diagnostics.snap` was the old file, and `update-snapshot.ts` doesn't exist.

Reply:
good catch, thanks